src/components/common/applicant-info-2.vue

```diff
@@ -116,31 +120,7 @@
           </v-tooltip>
         </v-col>
         <v-col cols="12" md="2" lg="2" />
-        <!--CORP NUMBER-->
-        <v-col cols="12" md="5" lg="5" v-if="getShowCorpNum === CorpNumRequests.COLIN">
-          <v-tooltip top
-            content-class="top-tooltip"
-            transition="fade-transition"
-            :disabled="isMobile"
-          >
-            <template v-slot:activator="{ on }">
-              <div v-on="on">
-                <v-text-field :messages="messages['corpNum']"
-                              :rules="corpNumRules"
-                              :error-messages="corpNumError"
-                              @focus="corpNumError = ''"
-                              :loading="loading"
-                              filled
-                              :label="corpNumFieldLabel"
-                              v-model="corpNum">
-                </v-text-field>
-              </div>
-            </template>
-            <span>
-              Enter the BC incorporation number of your business.
-            </span>
-          </v-tooltip>
-        </v-col>
```

Comment:
Removing this moves the new block (trademark) to the left but I think that's OK. Eg,

![image](https://github.com/bcgov/namerequest/assets/10002889/98078b20-7aca-4e88-b11d-350209240e5d)

Ditto in applicant-info-3.vue.